src/Workspaces/CoreTestUtilities/Formatting/FormattingTestBase.cs

```diff
@@ -46,7 +48,7 @@ public abstract class FormattingTestBase
 
                 var document = project.AddDocument("Document", SourceText.From(code));
 
-                var syntaxTree = await document.GetSyntaxTreeAsync();
+                var syntaxTree = (await document.GetSyntaxTreeAsync())!;
```

Comment:
We've got GetRequiredSyntaxTreeAsync().

Reply:
Fixed this and `GetProject`&rarr;`GetRequiredProject`